### DISCUSSION TITLE
Compatible with netdata

### DISCUSSION THREAD

**Discussion by ezequielsousabr** [netdata/netdata#14018] [Q&A] [answered]
I use netdata on several computers, but I don't know if it supports mobile. Is there a way to track these systems through netdata?

**Comment by Dim-P** [comment, marked as answer]
Hi @ezequielsousabr . I am not sure I understand the question, if you are asking whether you can install Netdata on mobile (e.g. Android) I am afraid this is not possible at the moment.

If your question is about accessing the agent dashboard on your mobile, you can do that (providing you are on the same local network) or, better, [you can claim your Netdata nodes on the cloud](https://learn.netdata.cloud/docs/agent/claim) and access the cloud from your mobile phone browser.

**Reply by ezequielsousabr** [reply]
Can you suggest some software that does this at the same time on multiple devices? Routers, Androids, Computers?

**Reply by Dim-P** [reply]
Not sure about a universal solution, Netdata can do routers, PCs, servers etc..
